src/reanimated2/animation/util.ts

```diff
@@ -43,6 +44,31 @@ if (__DEV__ && IS_REDUCED_MOTION) {
   );
 }
 
+const SHOULD_BE_USE_WEB = shouldBeUseWeb();
+
+export function assertEasingIsWorklet(
+  easing: EasingFunction | EasingFunctionFactory
+) {
+  'worklet';
+
+  if (SHOULD_BE_USE_WEB) {
+    /* It is possible to run reanimated on web without plugin, so let's skip this check on web */
+    return true;
+  }
+  const isFunction: boolean = typeof easing === 'function';
+  const functionName = typeof easing === 'function' ? easing?.name : '';
+  /* Just checking with `isWorkletFunction` is not sufficient in case of keyframe animations. Therefore, we also check for binding. */
+  const isBound = functionName.startsWith('bound');
```

Comment:
Lets try to avoid this check by fixing the Keyframe easing.

Reply:
It was fixed with the following line
```tsx
    functionInstance.__workletHash = workletHash;
```
Now function `isWorkletFunction` returns true for bound worklets as well